### DISCUSSION TITLE
Korath Easter Egg

### DISCUSSION THREAD

**Discussion by Ricky-Blackman** [endless-sky/endless-sky#10366] [Lore]
Sorry y'all, I'm kinda drunk, so ignore and delete this bs if required.
I am a listener to a popular podcast, which includes among its hosts a Jamaican-American man. His mother, being very religious, makes up her own swear words, including the famous, "Ringoseal! Bumbaclot!"
I would greatly appreciate it if the Korath (or another xeno race) had these words (ideally together) in the list of hails, or whatever you call it.

**Comment by RisingLeaf** [comment]
thats not really worth a discussion, make a PR or issue if you want this, but given this background ehhhh
~~I am for once taking the freedom top close this, as suggested by the author~~
